servicebroker/lib/util/validateUtil.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+module.exports = {
```

Comment:
This is too generic. It would probably be quicker just to write what we expect which is why it is best placed in the settings.validate method.

if (x != null) will catch both undefined and null so doing isUndefined and isNull isn't needed